includes/class-activitypub.php

```diff
@@ -123,7 +123,7 @@ public static function uninstall() {
 	 * @return string The new path to the JSON template.
 	 */
 	public static function render_activitypub_template( $template ) {
-		if ( defined( 'REST_REQUEST' ) && REST_REQUEST ) {
+		if ( wp_is_serving_rest_request() || wp_doing_ajax() ) {
```

Comment:
I'm afraid `wp_is_serving_rest_request()` is not compatible with our min WP version of 6.4. We could add a compat for it?